_includes/includes/ui/keyboard-details.php

```diff
@@ -476,11 +481,11 @@ protected static function WriteDownloadBoxes() {
         }
       }
 
-      if ($webtext) {
-        echo "<h2 class='red underline'>Online tools</h2>" . $webtext;
-      }
-
       if ($embed == 'none') {
+        if ($webtext) {
```

Comment:
I opted to make the "use online" link visible only on the website, rather than also in the app, where it is a distraction.